### DISCUSSION TITLE
help me understand)

### DISCUSSION THREAD

**Discussion by KantorSerhiy** [EvgSkv/logica#240] [Q&A] [answered]
Hello!
help solve the question(

I'm new to Logica and I'm wondering how I can make a sample like in the photo.

I can't understand at all.
I will be grateful for your kind advice!
![screen](https://user-images.githubusercontent.com/113110926/227660234-34a99b15-f435-4004-8ebc-0fffff2f6aba.jpg)

**Comment by EvgSkv** [comment, marked as answer]
Hello @KantorSerhiy !

Is it that for each name you want to find on which years it was the most popular?

If so, then assuming you have a predicate `PopularAt(name:, year:)` then a table like in the photo can be obtained with a rule:
```
NameToYears(name) List= year :- PopularAt(name:, year:);
```

If you are trying to achieve something else, please let me know, I'll be happy to help.

Thanks for trying out Logica!
Evgeny.

**Comment by KantorSerhiy** [comment]
hi!
it working and every day the Logica is more interesting to me!)


thank you for your reply, it is very important for me.

**Reply by EvgSkv** [reply]
Great to hear that! Happy to help when you have further questions. :-)
